manifests/05-deployment.yaml

```diff
@@ -28,3 +28,7 @@ spec:
                 fieldPath: metadata.namespace
           - name: OPERATOR_NAME
             value: "cluster-logging-operator"
+          - name: REPO_PREFIX
```

Comment:
I'm not sure this is something we want to always have for normal deployments

Reply:
I have no idea how it is supposed to work for normal deployments

Reply:
Scratch this comment, I thought I remembered that the e2e tests also created the deployment within the test. I had forgotten that it relies on this file.